zone/lua_client.cpp

```diff
@@ -1509,16 +1509,36 @@ bool Lua_Client::HasSpellScribed(int spell_id) {
 	return self->HasSpellScribed(spell_id);
 }
 
-void Lua_Client::SetAccountFlag(std::string flag, std::string val) {
+void Lua_Client::ClearAccountFlag(const std::string& flag) {
 	Lua_Safe_Call_Void();
-	self->SetAccountFlag(flag, val);
+	self->ClearAccountFlag(flag);
 }
 
-std::string Lua_Client::GetAccountFlag(std::string flag) {
+void Lua_Client::SetAccountFlag(const std::string& flag, const std::string& value) {
+	Lua_Safe_Call_Void();
+	self->SetAccountFlag(flag, value);
+}
+
+std::string Lua_Client::GetAccountFlag(const std::string& flag) {
 	Lua_Safe_Call_String();
 	return self->GetAccountFlag(flag);
 }
 
+luabind::object Lua_Client::GetAccountFlags(lua_State* L) {
+	auto t = luabind::newtable(L);
+	if (d_) {
+		auto self = reinterpret_cast<NativeType*>(d_);
+		auto l = self->GetAccountFlags();
+		auto i = 1;
```

Comment:
It feels a bit silly to have a simple type be `auto` - let's please just use the types for simple types it's easier to read

Reply:
For just `i` here?